### PR TITLE
feat(crowdfundings): Migrate memberships, add column gracePeriodInterval

### PR DESCRIPTION
**Pull request by patrickvenetz**
Replaces `UNCANCELLED_GRACE_PERIOD_DAYS` with `gracePeriodInterval` attribute on `memberships` table. Defaults to 14 days.

- [ ] Extend grace interval for recipients of a payment slip

### CODE REVIEW

File: servers/republik/modules/crowdfundings/graphql/resolvers/Membership.js [hunk 1]
```diff
@@ -91,8 +90,12 @@ module.exports = {
           return null
         }
 
-        return moment(getLastEndDate(periods))
-          .add(UNCANCELLED_GRACE_PERIOD_DAYS, 'days')
+        const graceEndDate = moment(getLastEndDate(periods))
+        Object.keys(membership.gracePeriodInterval).forEach(key => {
```
[the remark this answers is not in the document]
Comment (patte) [reply servers/republik/modules/crowdfundings/graphql/resolvers/Membership.js]: Thinking about it now that I read all the code, I think a small lib to add/subtract pg-intervals to moments would be handy.

[the remark this answers is not in the document]
Comment (patrickvenetz) [reply servers/republik/modules/crowdfundings/graphql/resolvers/Membership.js]: Done so, see https://github.com/orbiting/backends/pull/201

File: servers/republik/migrations/sqls/20190125150100-memberships-grace-period-interval-up.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "memberships"
+  ADD COLUMN "gracePeriodInterval" interval NOT NULL DEFAULT '14 days'::interval
```
[the remark this answers is not in the document]
Comment (patrickvenetz) [reply servers/republik/migrations/sqls/20190125150100-memberships-grace-period-interval-up.sql]: Renamed.

**Comment by patrickvenetz** [comment]
@patte Please revisit.

- Renamed `gracePeriodInterval` to `graceInterval`
- Replaced whitch-crafty Postgres interval add with `applyPgInterval` util

**Pull request merged**